pages/assessment.tsx

```diff
@@ -50,10 +52,14 @@ const Assessment: NextPage = () => {
     'let - block scope. Dersom variabelen blir deklarert med let i en funksjon, er den bare tilgjengelig i funksjonen. var - global scope Dersom variabelen blir deklarert med var, blir den tilgjengelig i all kode. Kan by på problemer når vi gir variabler samme navn.';
 
   const allAnswers: AnswerType[] = insperaDataToTextboxObject(data, taskNumber);
-  const answers = allAnswers.slice(0,10);
+  const answers = allAnswers.slice(0, 10);
   const numberOfAnswers = answers.length;
 
-  const p = answers.map((answer: AnswerType) => ({ score: '', isFlagged: false, ...answer }));
+  const p = answers.map((answer: AnswerType) => ({
```

Comment:
Prettier from here to line 234. (Arrggghhhh 😠  sometimes I feel that prettier just goes back and forth)